src/main/java/dev/emi/trinkets/api/TrinketsApi.java

```diff
@@ -21,16 +15,20 @@
 import net.minecraft.tag.Tag;
 import net.minecraft.util.Identifier;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
 public class TrinketsApi {
 
 	public static final ComponentKey<TrinketComponent> TRINKET_COMPONENT = ComponentRegistryV3.INSTANCE
 			.getOrCreate(new Identifier(TrinketsMain.MOD_ID, "trinkets"), TrinketComponent.class);
 	private static final Map<Identifier, Function3<ItemStack, SlotReference, LivingEntity, TriState>> QUICK_MOVE_PREDICATES
-			= new HashMap<Identifier, Function3<ItemStack, SlotReference, LivingEntity, TriState>>(); 
```

Comment:
Same question here as before, what is erasing generics doing other than reducing line size?